terraso_backend/apps/soil_id/models/soil_data.py

```diff
@@ -36,3 +84,15 @@ class SoilData(BaseModel):
         null=True, validators=[MinValueValidator(0), MaxValueValidator(100)]
     )
     slope_shape = models.CharField(null=True, choices=SLOPE_SHAPES)
+    slope_landscape_position = models.CharField(null=True, choices=LANDSCAPE_POSITIONS)
+    slope_aspect = models.IntegerField(
+        null=True, validators=[MinValueValidator(0), MaxValueValidator(359)]
+    )
+    slope_range_select = models.CharField(null=True, choices=SLOPE_RANGES)
+    slope_range_manual = models.IntegerField(
```

Comment:
suggestion: i think slope range and steepness are the same thing, so there is no need for `slope_range_manual`, and `slope_range_select` could be renamed to `slope_steepness_select`